DataAccess/CMakeLists.txt

```diff
@@ -1,8 +1,9 @@
 project(DataAccess LANGUAGES CXX)
 
 add_subdirectory(Public)
-add_subdirectory(DataAccess.Static)
 add_subdirectory(DataAccess.Debug)
+add_subdirectory(DataAccess.Static)
```

Comment:
Maybe then rename to DataAccess.Postgre?

Reply:
yeah, makes sense

Reply:
done.